Formula/o/odin.rb

```diff
@@ -81,6 +81,32 @@ def install
           "-I#{Formula["raylib"].include}", "-L#{Formula["raylib"].lib}", "-lraylib"
         cp "libraygui.dylib", buildpath/raylib_installpath
       end
+    else
+      raylib_installpath = "vendor/raylib/linux"
+
+      # Create symbolic links for GLFW and Raylib libraries
+      ln_s Formula["glfw"].lib/"libglfw3.a", buildpath/"vendor/glfw/lib/darwin/libglfw3.a"
+      ln_s Formula["raylib"].lib/"libraylib.a", buildpath/raylib_installpath/"libraylib.a"
+
+      # Adjust the name of the Raylib dynamic library for Linux
+      ln_s Formula["raylib"].lib/"libraylib.so.4.5.0", buildpath/raylib_installpath/"libraylib.so.500"
+
+      resource("raygui").stage do
+        # Copy the raygui header and source files
+        cp "src/raygui.h", "src/raygui.c"
+
+        # Build the static library for raygui
+        system ENV.cc, "-c", "-o", "raygui.o", "src/raygui.c",
+               "-fpic", "-DRAYGUI_IMPLEMENTATION", "-I#{Formula["raylib"].include}"
+        system "ar", "-rcs", "libraygui.a", "raygui.o"
+        cp "libraygui.a", buildpath/raylib_installpath
```

Comment:
This is identical to the macOS codepath and should be moved outside of the `if`-`else` block.